code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -53,6 +50,22 @@
 
 	do_evolve(castetype, castepick, TRUE)
 
+/mob/living/carbon/xenomorph/proc/get_evolution_options()
+	switch(tier)
+		if(XENO_TIER_ZERO)
+			return GLOB.xeno_types_tier_one
+		if(XENO_TIER_ONE)
+			return GLOB.xeno_types_tier_two + GLOB.xeno_types_tier_four
+		if(XENO_TIER_TWO)
+			return GLOB.xeno_types_tier_three + GLOB.xeno_types_tier_four
+		if(XENO_TIER_THREE)
+			return GLOB.xeno_types_tier_four
+		if(XENO_TIER_FOUR)
+			if(isxenoshrike(src))
+				return GLOB.xeno_types_tier_shrike
+			else
+				return list()
+
```

Comment:
use early returns: queen and sking return null instead of list() like the fail state